packages/apollo-codegen-swift/src/language.ts

```diff
@@ -29,7 +29,16 @@ export interface Property {
 }
 
 export function escapedString(string: string) {
-  return string.replace(/"/g, '\\"').replace(/\n/g, "\\n");
```

Comment:
FWIW, this escaping won't work with multiline strings either, because `\` is not special in a multilline string except as part of the token `\"""`.

Reply:
I've opened #1450 on handling of multiline strings in the Swift codegen.

To be honest, I have a longer-term plan to move the generation of codegen to Swift so it's easier for Swift developers to contribute. Since failure to handle multiline strings isn't a regression, I'd prefer to wait to address this issue until I'm working on the updated codegen, since I have a *way* better handle on Swift's string handling than Typescript's.

Reply:
Actually, come to think of it, the old code probably worked fine after all, because `string` presumably contains the properly-interpreted contents of the multiline string. In that case, this PR will explicitly break multiline strings.

I would strongly encourage you to add a test case that has a multiline string with backslash-escapes in it (since backslash-escapes, except for `\"""`, aren't meaningful in multiline strings), verify its behavior *without this PR*, and then see what this PR does to it.

Reply:
I snuck a couple additional tests in on #1452 to validate this. I'll try to get this being less cranky in the morning.